ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -6,7 +6,7 @@
   import { Icon } from "../Primitive";
   import Hoverable from "./Hoverable.svelte";
 
-  export let disabled: boolean = false;
+  export let blocked: boolean = false;
```

Comment:
This change doesn't appear to add much in my view. We use "disabled" more often across the project so reverting this rename wouldn't be a bad idea.